src/schema/imported/manifest.json

```diff
@@ -189,11 +189,18 @@
                       "properties": {
                         "extension_pages": {
                           "type": "string",
-                          "format": "contentSecurityPolicy"
+                          "format": "contentSecurityPolicy",
+                          "description": "The Content Security Policy used for extension pages."
                         },
                         "content_scripts": {
                           "type": "string",
-                          "format": "contentSecurityPolicy"
+                          "format": "contentSecurityPolicy",
+                          "description": "The Content Security Policy used for content scripts."
+                        },
+                        "isolated_world": {
```

Comment:
Wait do we have the code supporting this ? We have some validation code in the linter and some permission extraction code in addons-server that deal with content scripts, if we need to support `isolated_world` as an alias then we need some extra code in both to do it.

Reply:
Indeed, I must have missed that. I'll add issues in both, addons-server and addons-linter but I'm not quite sure if this blocks the actual schema import as we kinda need that first to have the linter and server process that value further.

Reply:
Actually, once the linter issues the correct warnings for the alias the code in addons-server will continue to work too as it's listening on the addons-linter internal codes which should stay the same (`MANIFEST_CSP`)

Reply:
Actually, I'm confused by this as https://developer.chrome.com/extensions/content_scripts#declaratively is clearly mentioning `content_scripts`. cc @rpl who might know more

Reply:
Alright, I talked to Luca about this and I'll try to summarize:

`isolated_worlds` is an alias for `content_scripts` *inside* the `content_security_policy` object, which historically only contained strings but can now be an object as well.

That means we're going to have to update a few checks in the linter.

I'll update the relevant issue #3053 accordingly and that then means that we need that to land along side with this schema import to avoid missing any custom content security policies.